src/test/java/com/puppycrawl/tools/checkstyle/grammar/GeneratedJavaTokenTypesTest.java

```diff
@@ -746,7 +752,7 @@ public void testTokenNumbering() {
                         + " 'GeneratedJavaTokenTypesTest' and verified"
                         + " that their old numbering didn't change")
             .that(tokenCount)
-            .isEqualTo(235);
+            .isEqualTo(237);
```

Comment:
Nothing to do with your changes, but why is the count off by 12? Maybe we should add a note about missing numbers.

Reply:
It's mostly in the test method below.

Reply:
You referring to the 10 tokens in https://github.com/checkstyle/checkstyle/blob/b54de1e5de66de289ee52f6ad60067a116264c4e/src/test/java/com/puppycrawl/tools/checkstyle/grammar/GeneratedJavaTokenTypesTest.java#L765 ?

Reply:
That, plus there’s at least one “placeholder” token in the lexer that’s had to take the place of some autogenerated tokens in ANTLR2  that don’t exist anymore. Let me find some links.

Reply:
This can be dropped if you haven't found any links. It could be good to just add this documentation in the code to be clearer, but it isn't required.

Reply:
Updated the PR with a minor commit to improve this whole test.